.github/workflows/performance.yml

```diff
@@ -33,6 +33,13 @@ jobs:
               run: |
                   npm ci
 
+            - name: Install specific versions of the themes used in tests
+              run: |
+                  npm run wp-env start
+                  npm run wp-env -- run tests-cli "wp theme update twentytwentyone --version=1.7"
+                  npm run wp-env -- run tests-cli "wp theme update twentytwentythree --version=1.0"
```

Comment:
Will these downgrade the theme if it has a more recent version?

Reply:
Yes, the idea was to keep the theme version consistent across the tests so the themes couldn't influence the results. It's unlikely, but I think it's still good to keep things as consistent as possible.

Reply:
Yeah. The rationale for this is to be able to control the environment so that only the Gutenberg PR affects the metrics. At some point, we should manually update the themes, but at least this will be an intentional change and won't affect any unrelated PR.